yeetfile.js

```diff
@@ -24,4 +24,49 @@ $`npm run assets`;
             file.install("./run/default.env", `${etc}/default.env`);
         },
     }));
+});
+
+// NOTE(Xe): Fixes #217. This is a "half baked" tarball that includes the harder
+// parts for deterministic distros already done. Distributions like NixOS, Gentoo
```

Comment:
NixOS has no problem with that. It only needs two derivations to build the go and node bits. Also generally upstream tarballs are avoided because they sometimes have problems like patches don't apply or they are a commit off due to release problems.